scripts/test-docker

```diff
@@ -15,12 +15,12 @@ done
 
 test_docker_versions["ubuntu:18.04"]="^19.03.15$|^20.10"
 test_docker_versions["ubuntu:20.04"]="^^19.03.15$|^20.10"
-test_docker_versions["ubuntu:22.04"]="^20.10.13$|^20.10.1[3-9]"
+test_docker_versions["ubuntu:22.04"]="^20.10.13$|^20.10.1[3-9]|^20.10.2[0-3]"
```

Comment:
I suggest we update the ranges here to include all 20.10.2x versions ahead of time - it will be easy to forget to update test range every time we introduce new docker patch.

Also, I believe `^20.10.1[3-9]` includes `20.10.13` so there is no need for explicit `^20.10.13$` in regex.

Let's also extend the range for the other 2 OSes below to include all 20.10.2x versions 

All this said, I would change the range as below.
```suggestion
test_docker_versions["ubuntu:22.04"]="^20.10.1[3-9]|^20.10.2[0-9]"
```

Reply:
Ah! That's right. Updated the range to `^20.10.2[0-9]` for ubuntu 22.04, centos 8 and rockylinux 8